explorer/models/azureRegistryNodes.ts

```diff
@@ -291,12 +300,12 @@ export class AzureLoadingNode extends NodeBase {
     }
 }
 
-async function acquireToken(session: AzureSession): Promise<{ accessToken: string; refreshToken: string; }> {
+async function acquireToken(session: AzureSession): Promise<any> {
```

Comment:
What is the point to changing the type here? Additionally I think Julia made a unified location for this method.